pipelines/cli.go

```diff
@@ -0,0 +1,62 @@
+package pipelines
+
+import (
+	corecommon "github.com/jfrog/jfrog-cli-core/v2/docs/common"
+	"github.com/jfrog/jfrog-cli/utils/cliutils"
+	"github.com/urfave/cli"
+)
+
+func GetCommands() []cli.Command {
+	return cliutils.GetSortedCommands(cli.CommandsByName{
+		{
+			Name:         "status",
+			Flags:        cliutils.GetCommandFlags(cliutils.Status),
+			Aliases:      []string{"s"},
+			Description:  "Get status of latest run of pipeline",
```

Comment:
Please add Usage, HelpName, UsageText, and ArgsUsage exactly as we do in artifactory/cli.go. There is a good reason for each one of these fields.
```suggestion
```